### PR TITLE
[ERC20] Add 94 tokens for mainnet

### PR DESCRIPTION
**Pull request by bh2smith**
Its been a while since we have done an update here... so here are 94 new tokens. Results coming from this [V1 Query](https://dune.com/queries/1317323)

### CODE REVIEW

**Review by meththot — APPROVED** [review]

**Comment by Tomek00177** [comment]
Jejsj

**Review by meththot — APPROVED** [review]
Thanks

**Review by meththot — APPROVED** [review]

**Review by meththot — APPROVED** [review]

**Review by meththot — APPROVED** [review]

**Comment by meththot** [comment]
taks

File: deprecated-dune-v1-abstractions/ethereum/erc20/tokens.sql
Comment (meththot) [inline deprecated-dune-v1-abstractions/ethereum/erc20/tokens.sql:1]: thanks

**Review by meththot — APPROVED** [review]

**Review by Tomek00177 — APPROVED** [review]

**Comment by Tomek00177** [comment]
Tomi

**Review by Tomek00177 — APPROVED** [review]

**Review by Tomek00177 — APPROVED** [review]

**Comment by bh2smith** [comment]
This doesn't appear to be happening... the missing list is over 250 by now, so I will close this as it is outdated.

**Pull request closed without merging**